src/interfaces/Interfaces.tsx

```diff
@@ -0,0 +1,20 @@
+export default interface ServerlessApp {
+    app_id: string;
+    name: string;
+    version: string;
+    author: string;
+    tagline: string;
+    logo_url: string;
+    description: string;
+    tags: string[];
+    maintainer: string;
+    links: {
```

Comment:
This field isn't currently in [the app API data model spec](https://github.com/knative-scout/app-api/blob/master/DESIGN.md#data-model).

Should we add it to the spec?

Reply:
on second thought we can cut it from here. If we have the repository link, they can put whatever other links in their markdown. 
Also yes styling is WIP. It seems to not be fully loading patternfly default styling rn which is strange but I'll figure it out.

Reply:
Okay cool, lgtm